tabs/pid_tuning.js

```diff
@@ -414,20 +414,27 @@ TABS.pid_tuning.initialize = function (callback) {
         curveContext.lineTo(width / 2, height);
         curveContext.stroke();
 
-        if (scaleHeight <= height / 2) {
-            curveContext.strokeStyle = '#c0c0c0';
-            curveContext.lineWidth = 4;
+    }
 
-            curveContext.beginPath();
-            curveContext.moveTo(0, height / 2 + scaleHeight);
-            curveContext.lineTo(width, height / 2 + scaleHeight);
-            curveContext.stroke();
 
+    function plotStickPosition(curveContext, rcData, rate, rcRate, rcExpo, superExpoActive, deadband, maxAngularVel, stickColor) {
+
+        const DEFAULT_SIZE = 60; // canvas units, relative size of the stick indicator (larger value is smaller indicator)
+        const rateScaling  = (curveContext.canvas.height / 2) / maxAngularVel;
+
+        var currentValue = self.rateCurve.rcCommandRawToDegreesPerSecond(rcData, rate, rcRate, rcExpo, superExpoActive, deadband);
+
+        if(rcData!=undefined) {
+            curveContext.save();
+            curveContext.fillStyle = stickColor || '#000000';
+
+            curveContext.translate(curveContext.canvas.width/2, curveContext.canvas.height/2);
             curveContext.beginPath();
-            curveContext.moveTo(0, height / 2 - scaleHeight);
-            curveContext.lineTo(width, height / 2 - scaleHeight);
-            curveContext.stroke();
+            curveContext.arc(rcData-1500, -rateScaling * currentValue, curveContext.canvas.height / DEFAULT_SIZE, 0, 2 * Math.PI);
+            curveContext.fill();
+            curveContext.restore();
         }
+        return currentValue.toFixed(0); // The calculated value in deg/s is returned from the function call for further processing.
```

Comment:
Rounding this way has the undesired side effect that if the rc input is jittering between 0.5 and -0.5, the value that is displayed will jitter between 0 and -0. Maybe add `abs(currentValue) < 0.5 ? 0 : currentValue.toFixed` (or do similar before using the value).